### PR TITLE
Added altair_pandas

### PR DESCRIPTION
**Pull request by thewchan**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

This package is not currently on PyPI yet, so it is packaged via the github archive.

### CODE REVIEW

File: recipes/altair_pandas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+source:
+  url: https://github.com/altair-viz/{{ name }}/archive/master.tar.gz
+  sha256: 5248f8a94562ffa30e277051ab9dc9a487269a08b57d9e9f4a7b921e6ed5d884
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```
Comment (xhochy) [inline recipes/altair_pandas/meta.yaml:22]:
```suggestion
```

File: recipes/altair_pandas/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
```
Comment (xhochy) [inline recipes/altair_pandas/meta.yaml:5]:
```suggestion
```

File: recipes/altair_pandas/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+{% set name = "altair_pandas" %}
+{% set version = "0.1.0dev0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/altair-viz/{{ name }}/archive/master.tar.gz
```
Comment (xhochy) [inline recipes/altair_pandas/meta.yaml:14]: We can only package this once there has been a release.

**Review by xhochy — CHANGES_REQUESTED** [review]

File: recipes/altair_pandas/meta.yaml (hunk 3 above)
Comment (thewchan) [reply recipes/altair_pandas/meta.yaml:14]: Ok, I can leave this open for now? Or should I close it and wait for the release?

Comment (xhochy) [reply recipes/altair_pandas/meta.yaml:9]: You can leave it open, I justed wanted to add a ❌ so that we don't accidentially merge.

Comment (thewchan) [reply recipes/altair_pandas/meta.yaml:9]: Sounds good; thanks!

**Pull request closed without merging**